portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -37,8 +55,25 @@ public void init() {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
 
-    String json = convertToJson(comments);
+    for (Entity entity : results.asIterable()) {
```

Comment:
I love for..of loops like this that don't need a loop index. I find them much easier to follow. :)